clustering.Rmd

```diff
@@ -238,75 +294,93 @@ Note, however, that "large" is relative to *both* the scale of the variables for
 
 ### The clustering algorithm
 
-We begin the K-means \index{K-means!algorithm} algorithm by picking K, and uniformly randomly assigning data to the K clusters. 
+We begin the K-means \index{K-means!algorithm} algorithm by picking K, 
+and randomly assigning an equal number of observations 
```

Comment:
We don't necessarily assign an equal # of obs to each cluster. We do typically uniformly assign labels tho. I would revert this change

Reply:
The reviewers said this was unclear (the uniformly part in particular), which was why I changed it...

How about a "randomly assigning a roughly equal number of observations"